source/install/install-kubernetes-cluster.rst

```diff
@@ -0,0 +1,36 @@
+.. _install-kubernetes-cluster:
+
+Set Up a Kubernetes Cluster
+============================
+
+If you do not already have a production-ready Kubernetes cluster you will need to set one up. If you already have a Kubernetes cluster you may start with step 2.
+
+Your Kubernetes cluster must be version 1.11 or higher.
```

Comment:
i think is better set 1.12
we are using some resources that exist in 1.12 (more stable)